@kiva/kv-activity-feed/src/utils/parseError.ts

```
@@ -0,0 +1,10 @@
+const parseError = (error: any) => {
+	if (error instanceof Error) {
+		console.error('Error creating StreamService instance:', error.message);
+	} else {
+		console.error('Error creating StreamService instance:', error);
+	}
+	throw new Error('Failed to create StreamService instance');
```

I'm guessing we don't want to re-throw the exception.